tekton/release-cheat-sheet.md

```diff
@@ -25,26 +32,26 @@ the pipelines repo, a terminal window and a text editor.
       - name: url
         value: https://github.com/tektoncd/pipeline
       - name: revision
-        value: # Example: 33e0847e67fc9804689e50371746c3cdad4b0a9d
+        value: # The commmit you selected in the last step, e.g. 33e0847e67fc9804689e50371746c3cdad4b0a9d
     ```
 
-4. `kubectl apply -f tekton/resources.yaml`
+1. Apply file you just made to the dogfooding cluster: `kubectl apply -f your-pipeline-resource-file.yaml`
 
-5. Create environment variables for bash scripts in later steps.
+1. Create environment variables for bash scripts in later steps.
 
     ```bash
-    TEKTON_VERSION=# Example: v0.11.2
-    TEKTON_RELEASE_GIT_RESOURCE=# Example: tekton-pipelines-v0-11-2
-    TEKTON_IMAGE_REGISTRY=gcr.io/tekton-releases
+    export TEKTON_VERSION=# Example: v0.11.2
+    export TEKTON_RELEASE_GIT_RESOURCE=# Name of the resource you created, e.g.: tekton-pipelines-v0-11-2
+    export TEKTON_IMAGE_REGISTRY=gcr.io/tekton-releases # only change if you want to publish to a different registry
```

Comment:
Which step further down failed when you didn't include the `export`?  FWIW I don't think I've ever had to use them and it shouldn't be necessary unless (for example) I opened a tmux after setting these envs, or otherwise somehow created a subprocess shell before using them.

Reply:
In my environment (zsh with iterm on a mac) a line like `TEKTON_RELEASE_GIT_RESOURCE="foo"` is only applicable for the running command (process?), e.g. something like `TEKTON_RELEASE_GIT_RESOURCE="foo" mycommand` but isn't retained for future commands unless exported, e.g.:

1. I run the command to set the var:
```
TEKTON_RELEASE_GIT_RESOURCE="foo"
```

2. I run `env` to see what environment variables are set, it's not listed

3. I export it instead:
```
 export TEKTON_RELEASE_GIT_RESOURCE="foo"
```

4. Now when I run `env` I see it set:
```
env
...
TEKTON_RELEASE_GIT_RESOURCE=foo
_=/usr/bin/env
```

I think this explains it pretty well: https://stackoverflow.com/questions/7328223/unix-export-command 

> If you set a variable at the command-line like
> $ FOO="bar"
> That variable will not be visible in child processes

I kinda wonder how it's working with your setup!

Reply:
I'm running the same shell and OS.  It's true that it won't show up with `env` because `env` only shows variables marked for export.  Run `set` and you'll see it listed.

Here's a tiny example to test it out:

```bash
$ FOO=bar
$ echo $FOO
```

This prints "bar" for me.

Reply:
ahhh okay i think im starting to understand what's happening - i think im explaining what you already understand, but for my own understanding: so if you say `FOO=bar`, it will be set for the current process (TIL about `set`) - if you wanted FOO to be set for any child processes,  you'd need to use `export`

```bash
# set BAR in the current environment/process
✗ BAR=baz
✗ echo $BAR
baz

# we have a script that tries to use the same variable
 ✗ cat foo.sh
set -x

echo $BAR

# the variable isn't exported, so when this is run, a child process starts and is not provided with that variable
✗ ./foo.sh
+ echo

# if we export the variable, it will be provided to the child process
✗ export BAR=hello
✗ ./foo.sh
+ echo hello
hello
```

And in the cheatsheet, we're resolving the variables in the shell that runs the commands, so no need to pass them to any child processes - so I think I'm on board now with not including `export` - lemme know if my understanding isn't correct @sbwsg !

Reply:
I think I've made this same change in other docs 🤦‍♀️  I'll try to reverse it when I see it. Thanks again @sbwsg , I feel like my overall understanding is deeper now :D